File: C/BOJ_2444_별 찍기 - 7.c
# include <stdio.h>

int main()
{
	int num;
	scanf("%d", &num);
	for (int i = 0; i < num; i++) {
		for (int j = 0; j < num - i - 1; j++) printf(" ");
		for (int k = 0; k < (i * 2) + 1; k++) printf("*");
		printf("\n");
	}
	
	for (int i = 1; i < num; i++) {
		for (int k = 0; k < i; k++) printf(" ");
		for (int j = 0; j < ((num - i) * 2) - 1; j++) printf("*");
		printf("\n");
	}
	return 0;
}

File: Python/BOJ_2444_별 찍기 - 7.py
import sys

num = int(sys.stdin.readline())
count = 1
while count <= num:
    print(' ' * (num - count), end = '')
    print('*' * (1 + (2 * (count - 1))))
    count += 1

count -= 2
while count > 0:
    print(' ' * (num - count), end = '')
    print('*' * (1 + (2 * (count - 1))))
    count -= 1